app/controllers/provider_foreman_controller.rb

```diff
@@ -368,27 +341,23 @@ def tree_select_unprovisioned_configured_system
   def configuration_manager_providers_tree_rec
     nodes = x_node.split('-')
     case nodes.first
-    when "root" then rec = find_record(ManageIQ::Providers::ConfigurationManager, params[:id])
+    when "root" then find_record(ManageIQ::Providers::ConfigurationManager, params[:id])
```

Comment:
@kbrock It buys us a little bit dealing with compressed ids.  But I'd like to say that is out of scope for this PR.